orb/core/connection_types/sql/postgres/psqlconnection.py

```diff
@@ -16,17 +16,24 @@
 
 try:
     import psycopg2 as pg
+    import psycopg2.extensions as pg_ext
+
     from psycopg2.extras import DictCursor, register_hstore, register_json
-    from psycopg2.extensions import QueryCanceledError
 
 except ImportError:
     log.debug('For PostgreSQL backend, download the psycopg2 module')
 
-    QueryCanceledError = orb.errors.DatabaseError
```

Comment:
used with the module import now `pg_ext.QueryCanceledError`.  It doesn't need to be explicitly defined, I only ever had it in there as a linter satisfaction, one of those "QueryCanceledError could be not defined in XYZ scenario" complaints